artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/components/consumers.js

```diff
@@ -204,7 +204,7 @@ var Artemis;
             Artemis.log.debug("navigating to consumer = " + artemisConsumer.consumer.session);
```

Comment:
Should we change the log too?
```suggestion
            Artemis.log.debug("navigating to consumer = " + artemisConsumer.consumer.sessionID);
```

Reply:
<humble-voice>of course!

Reply:
updated the PR (squashed the additional changes)